src/algebra/group_power.lean

```diff
@@ -635,11 +634,31 @@ lemma pow_le_one {x : α} : ∀ (n : ℕ) (h0 : 0 ≤ x) (h1 : x ≤ 1), x ^ n 
 end linear_ordered_semiring
 
 theorem pow_two_nonneg [linear_ordered_ring α] (a : α) : 0 ≤ a ^ 2 :=
-by rw pow_two; exact mul_self_nonneg _
+by { rw pow_two, exact mul_self_nonneg _ }
+
+/-- Bernoulli's inequality for `n : ℕ`, `0 ≤ 2 + a` -/
+theorem one_add_mul_le_pow [linear_ordered_ring α] {a : α} (H : 0 ≤ 2 + a) :
```

Comment:
I think the hypothesis `-2 ≤ a` easier to work with.

Reply:
Done.